style.css

```
@@ -0,0 +1,241 @@
+    background-color: rgba(60, 16, 216, 0.5);
+    position: sticky;
+    top:0;
+}
+
+.navbar ul{
+    display: flex;
+    list-style: none;
+    margin: 20px 0px;
+}
+
+.navbar ul li{
+    font-family: century;
+    font-size: 1.1rem;
+    font-weight: bold;
+}
+
+.navbar ul li a{
+    text-decoration: none;
+    color:white;
+    padding: 8px 25px;
+    transition: all .5s ease;
+}
+
+.navbar ul li a:hover{
+    background-color: white;
+    color: black;
+    box-shadow: 0 0 10px white;
+}
+
+#home h1{
+    color: white;
+}
+
+#home::before{
+    content: "";
+    position: absolute;
+    top:0;
+    right:0;
+    background: url(img/index_back.jpeg) no-repeat center center/cover ;
+    height: 1000px;
+    width: 100%;
+    z-index: -1;
+    opacity: .8;
+    margin-bottom: 0;
+}
+
+.main{
+    border: 1px solid rgb(248, 245, 245);
+    position: absolute;
+    top:30%;
+    right: 10%;
+    left:10%;
+}
+
+.headings{
+    font-family: center;
+    font-size: 3rem;
+    text-align: center;
+    margin-top: 40px 0px;
+}
+
+/* <--about.html--> */
+#about{
+    margin-bottom: 0;
+    background-color:rgb(10, 231, 220);
+    height: 1000px;
+    width: 100%;
+}
+
+#pic{
+    display: flex;
+}
+
+#pic img{
+    margin-top: 20px;
+    width: 575px;
+    height: 400px;
+}
+
+.headings{
+    color:rgb(23, 26, 218);
+}
+
+#intro{
+    
+    padding:10px;
+}
+
+#intro h2{
+    font-size: 2rem;
+    margin-bottom: 20px;
+}
+
+#intro h3{
+    font-size: 1rem;
+    margin-bottom: 20px;
+}
+
+/* <--portfolio--> */
+#portfolio{
+    background-color: rgb(193, 236, 197);
+    height: 1000px;
+    width: 100%;
+}
+
+.project{
+    display: flex;
+    justify-content: center;
+    background-color:rgb(61, 204, 223);
+    position: sticky;
+    top:0;
+}
+
+.gallery{
+    border:1px solid rgb(139, 91, 91);
+    margin: 100px;
+    align-items:center;
+    text-align: justify;
+    padding: 10px;
+    border-radius: 15px; 
+    background-size: 100% 200%;
+    background-color: rgb(232, 235, 97);
+    transition: all .8s; 
+}
+.gallery p{
+    color:rgb(255, 71, 86);
+    font-size: 30px;
+}
+/* <--hobbies--> */
+#hobbies{
+    margin-top:0;
+    background-color: rgb(150, 231, 231);
+    height: 1000px;
+    width: 100%;
+}
+.my_hobbies{
+    display: flex;
+    justify-content: center;
+    background-color:rgb(73, 214, 214);
+    position: sticky;
+}
+
+.headings{
+    margin-top: 0px;
+    color: rgb(109, 128, 0);
+}
+.box p{
+    color:rgb(240, 238, 243);
+}
+.box{
+    width: 300px;
+    height: 450px;
+    border:1px soli yellowgreen;
+    margin: 10px;
+    align-items:center;
+    text-align: justify;
+    padding: 10px;
+    border-radius: 15px;
+    background-color: hotpink;
+    transition: all .8s;
+}
+
+.box img{
+    width: 200px;
+    height: 200px;
+    background-color: white;
+    padding: 10px;
+}
+/* <--contact section--> */
+#contact{
+    display: flex;
+    flex-direction: column;
+    box-shadow: border-box;
```

I _think_ this is reversed.

```suggestion
    border-box: box-shadow;
```